SwiftUIDZ/PlayerViewModel.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  PlayerViewModel.swift
+//  SwiftUIDZ
+//
+//  Created by 2lup on 18.01.2023.
+//
+
+import AVFoundation
+
+// Вью-модель для плеера
+class PlayerViewModel: ObservableObject {
+    // MARK: - Constants
+    
+    private enum Constants {
+        static let songNames = ["Lightwire - City of Dreams", "Lightwire - Over Powered", "Lightwire - Sunset"]
+        static let secText = "s"
+        static let minText = "m"
+        static let pointText = " : "
+        static let emptyText = ""
+        static let formatSongText = "mp3"
+        static let zeroNumberFloat: Float = 0.0
+        static let zeroNumberDouble: Double = 0.0
+        static let zeroNumberInt = 0
+        static let intervalTimerNumber = 0.2
+        static let hourNumber = 3600
+        static let minNumber = 60
+    }
+    
+    // MARK: - Public Properties
+    
+    @Published public var maxDuration: Float = Constants.zeroNumberFloat
```

Comment:
```suggestion
    @Published var maxDuration: Float = Constants.zeroNumberFloat
```
public пишем когда создаем свой Фреймворк